dashboard/modules/reporter/reporter_agent.py

```diff
@@ -116,6 +130,7 @@ def __init__(self, dashboard_agent):
         self._cpu_counts = (psutil.cpu_count(),
                             psutil.cpu_count(logical=False))
         self._ip = ray._private.services.get_node_ip_address()
+        self._redis_address, _ = dashboard_agent.redis_address
```

Comment:
add a new variable here `self.is_head_node = (self._ip == sslf._redis_address)` and used downstream

Reply:
good idea, done